airflow/models/baseoperator.py

```diff
@@ -834,7 +836,6 @@ def clear(self,
         Clears the state of task instances associated with the task, following
         the parameters specified.
         """
-        TI = TaskInstance
```

Comment:
OK I get it now why we had TI at the top :)

Reply:
Yeah. I will change it to TaskInstance everywhere. I prefer longer names.

Reply:
Me too :)